### ISSUE TITLE
What's the point of using php?

### ISSUE THREAD

**Issue by ArjixWasTaken** [KirixenYT/Gogoanime#11]
You are not actually using php, you only have it to split html in multiple files...

**Comment by ArjixWasTaken** [comment]
For example, this would be the perfect use of a for-loop in PHP

https://github.com/KirixenYT/Gogoanime/blob/main/anime-list.php#L69-L146

Sure, you do a for-loop for the actual content, but that is because it cannot be done in plain html
https://github.com/KirixenYT/Gogoanime/blob/main/anime-list.php#L149-L161

Also, what the fuck is that php code

```php
<?php
  $json = file_get_contents("$apiLink/animeList?page=$page");
  $json = json_decode($json, true);
  foreach($json as $animeList)  { 
?>
    
    <li title='<?php $desc = $animeList['liTitle']; echo htmlspecialchars($desc);?>'> <a href="/category/<?=$animeList['animeId']?>" title=""><?=$animeList['animeTitle']?></a></li>
<?php } ?>
```

you could make it easier to read you know

```php
<?php

$json = file_get_contents("$apiLink/animeList?page=$page");
$json = json_decode($json, true);

foreach($json as $animeList)  { ?>
    <li title='<?= htmlspecialchars($animeList['liTitle']) ?>'>
        <a href="/category/<?= $animeList['animeId'] ?>" title=""><?= $animeList['animeTitle'] ?></a>
    </li> <?php
}
```

**Comment by ArjixWasTaken** [comment]
yeah ok I was being mean, sorry about that

**Comment by ArjixWasTaken** [comment]
But....I did notice that you are loading the actual analytics scripts from gogocdn...
Did you just copy paste the html from the actual website?

**Comment by ruriescaper** [comment]
Nah, originally the whole script was unfinished from 2019 maybe, he's using
it as a base....▪️

**Comment by ArjixWasTaken** [comment]
so this is a fork?

**Comment by ruriescaper** [comment]
maybe you can say that, but he actually made many changes to make it work
....

**Comment by SailorSammyy** [comment]
> But....I did notice that you are loading the actual analytics scripts from gogocdn... Did you just copy paste the html from the actual website?

As what @ruriescaper said, I've only updated the website to work properly, and yes it is using the cdn from the actual gogoanime website and I believe the code was also copied from the actual website but with majority updates.. and sorry for the late reply, My github account got locked and I lost the access to it and yeah this is the new account im gonna use from now on!
